README.md

```diff
@@ -21,7 +21,11 @@
 
 # 📖 [AWESOME BOOKS ES6] <a name="about-project"></a>
 
+<<<<<<< HEAD
+**Awesome-books-ES6** is a project that we are applying our knowledge on HTML and CSS and Javascript to showcase our most crisp and pristine projects.
+=======
 **Awesome-books-ES6** is a project that we are applying our knowledge on HTML and CSS and Javascript to showcase our most crisp and pristine projects. 
+>>>>>>> 8756b8cfd46a3c8644ac5e60c29a5503f6792464
```

Comment:
- [ ] As I've mentioned previously please make sure to remove any text that doesn't add value to your readme check the screenshot below. 👇
  ![image](https://user-images.githubusercontent.com/104968236/235476224-048c2f8d-e77d-4209-aabe-c423481a03dc.png)